homeassistant/components/plex/server.py

```diff
@@ -39,12 +43,34 @@
 plexapi.X_PLEX_VERSION = X_PLEX_VERSION
 
 
+def debounce(func):
+    """Decorate function to debounce callbacks from Plex websocket."""
+
+    @callback
+    def call_later_listener(self, _):
+        """Handle call_later callback."""
+        self.debounce = None
+        self.hass.async_add_executor_job(func, self)
```

Comment:
Make this `async` and await the executor
```suggestion
    async def call_later_listener(self, _):
        """Handle call_later callback."""
        self.debounce = None
        await self.hass.async_add_executor_job(func, self)
```